build.sbt

```diff
@@ -29,27 +14,38 @@ val commonSettings: Seq[SettingsDefinition] = Seq(
 resolvers += Resolver.sonatypeRepo("releases")
 
 
-lazy val root = (project in file("."))
-  .aggregate(eventProducerPlayJson27, eventProducerPlayJson28)
-  .settings(
-    licenses += ("MIT", url("http://opensource.org/licenses/MIT")),// to keep bintrayEnsureLicenses happy
-    sourceDirectory := baseDirectory.value / "dummy",
-    publishArtifact := false,
-    bintrayReleaseOnPublish := false
-  )
-
-lazy val eventProducerPlayJson27 = (project in file("play27"))
+lazy val models = (project in file("models"))
   .settings(commonSettings: _*)
   .settings(
-    sourceDirectory := baseDirectory.value / "../src",
-    libraryDependencies ++= Seq("com.typesafe.play" %% "play-json" % "2.7.4"),
-    name := "acquisition-event-producer-play27"
+    addCompilerPlugin("org.scalamacros" % "paradise" % "2.1.1" cross CrossVersion.full), // for simulacrum
+    libraryDependencies ++= Seq(
+      "com.gu" %% "ophan-event-model" % "0.0.17" excludeAll ExclusionRule(organization = "com.typesafe.play"),
+      "org.typelevel" %% "simulacrum" % "1.0.0",
+      "com.gu" %% "fezziwig" % "1.3",
+      "io.circe" %% "circe-core" % "0.12.1",
+      "com.typesafe.play" %% "play-json" % "2.6.14",
+      "org.scalatest" %% "scalatest" % "3.1.1" % "test"
+    ),
+    name := "acquisition-event-models-play26"
   )
 
-lazy val eventProducerPlayJson28 = (project in file("play28"))
+lazy val client = (project in file("client"))
   .settings(commonSettings: _*)
   .settings(
-    sourceDirectory := baseDirectory.value / "../src",
-    libraryDependencies ++= Seq("com.typesafe.play" %% "play-json" % "2.8.1"),
-    name := "acquisition-event-producer-play28"
+    addCompilerPlugin("org.scalamacros" % "paradise" % "2.1.1" cross CrossVersion.full), // for simulacrum
+    libraryDependencies ++= Seq(
+      "ch.qos.logback" % "logback-classic" % "1.2.3",
+      "com.gu" %% "acquisitions-value-calculator-client" % "2.0.5",
+      "com.squareup.okhttp3" % "okhttp" % "3.9.0",
+      "com.typesafe.scala-logging" %% "scala-logging" % "3.9.2",
+      "org.typelevel" %% "simulacrum" % "1.0.0",
+      "org.scalatest" %% "scalatest" % "3.1.1" % "test",
+      "org.scalactic" %% "scalactic" % "3.1.1",
+      "org.typelevel" %% "cats-core" % "2.1.1",
+      "com.amazonaws" % "aws-java-sdk-kinesis" % "1.11.465",
+      "com.gu" %% "thrift-serializer" % "4.0.3"
```

Comment:
this is the key dependency for the client only, but actually quite a few things could be trimmed from the model